### ISSUE TITLE
neuer Naviknopf Design noch nicht eingebunden

### ISSUE THREAD

**Issue by helfjama** [iqb-berlin/verona-modules-aspect#702]
Der neue Naviknopf funktioniert, aber das Design des Knopfes ist noch nicht drin: 
<img width="726" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBN0J2Qnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--1b9c53e1f5d399391059976a97ed5576d1ab165d/grafik.png" alt="grafik.png" />
(siehe https://www.iqb-studio.de/#/a/438/65820/preview)

**Comment by helfjama** [comment]
Das Design ist jetzt da , aber der Button noch versetzt: 
<img width="950" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBLzJDQnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--52d3052f948a675531a7eb9fb558db66a719035f/grafik.png" alt="grafik.png" />
siehe: https://www.iqb-studio.de/#/a/438/65913/preview
So sollte es aussehen:
<img width="1045" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBLytDQnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--8f444a70d758ac0e8371a7e889479a4a25c071f8/grafik.png" alt="grafik.png" />

**Comment by jojohoch** [comment]
Problem in Firefox

**Comment by helfjama** [comment]
Ja, das stimmt - aber was kann man da machen? So sollte es ja möglichst nicht bleiben?

**Comment by rhenck** [comment]
Also das ist unseres Wissens geloest. Ist das aktuell bei dir noch ein Problem?

**Comment by helfjama** [comment]
Der Punkt fehlt noch und in Firefox zumindest ist der Button auch auf der 2,8 noch versetzt: 
<img width="316" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBL2FkQ0E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--75c17a1386d4036a91bfef6f7ac5f6344529f462/grafik.png" alt="grafik.png" />
